CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ set(CMAKE_CXX_STANDARD 11)
 
 project(visGUI)
 
+if(MSVC AND (MSVC_VERSION GREATER_EQUAL 1914))
+  # qtExtensions uses the __cplusplus macro to test for C++11 support
```

Comment:
This should be fixed by kitware/qtextensions@a1ebde053ccf71f60f1e6928ec04fea1aa025a28; are you sure you need this?

Reply:
The version of qtextensions I'm using has that commit. In building VIAME, however, it doesn't seem to use that flag when building vivia.

Reply:
That's... concerning. Does your `qtExtensionsConfigTargets.cmake` have exported compile options for `qtExtensionsHeaders`? (AFAIK it is working for other people...)

Reply:
It does not. Perhaps the problem is here, with the usage of `ADDED_FLAG`?:
https://github.com/Kitware/qtextensions/blob/a911d919bc8f01c6f3af067dc95a654b2f27cf16/cmake/config.cmake#L88

That variable is set using `PARENT_SCOPE`, but not by `qte_add_cxx_flags`, but rather its callee.

Reply:
Hmm, yes, that looks plausible. Care to submit a patch to qtExtensions? (I wonder how no one else has noticed this...)